scripts/standard_configure.bat

```diff
@@ -8,21 +8,32 @@ rem Optional parameters:
 rem You can find a list will all the compiation options in INSTALL.md or here:
 rem  - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options
 
-rem Set compiler
-set CC=cl.exe
-set CXX=cl.exe
-
 rem Set variables
+if not defined USE_INTEL_LLVM set USE_INTEL_LLVM="OFF"
 if not defined KRATOS_SOURCE set KRATOS_SOURCE=%~dp0..
 if not defined KRATOS_BUILD set KRATOS_BUILD=%KRATOS_SOURCE%/build
 
+rem oneAPI call and set compiler
+IF "%USE_INTEL_LLVM%"=="ON" (
+    call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"
+    set CC=icx.exe
+    set CXX=icx.exe
+    @REM Only Ninja compiler is supported with Intel LLVM
+    set CMAKE_GENERATOR=Ninja
+) ELSE (
+    set CC=cl.exe
+    set CXX=cl.exe
+)
+
 rem Warning: In windows this option only works if you run through a terminal with admin privileges
 rem set KRATOS_INSTALL_PYTHON_USING_LINKS=ON
 
 rem Set basic configuration
 if not defined KRATOS_BUILD_TYPE set KRATOS_BUILD_TYPE=Release
-if not defined BOOST_ROOT set BOOST_ROOT=C:\CompiledLibs\boost_1_67_0
+if not defined BOOST_ROOT set BOOST_ROOT=C:\CompiledLibs\boost_1_83_0
```

Comment:
I don't think boost should be changed in this PR.

Reply:
At the end depends of the developer, but that version is quite old